src/MongoDB.Driver/Encryption/MongocryptdFactory.cs

```diff
@@ -119,7 +119,7 @@ private bool ShouldMongocryptdBeSpawned(out string path, out string args)
                     path = string.Empty; // look at the PATH env variable
                 }
 
-                if (Directory.Exists(path))
+                if (Directory.Exists(path) || string.IsNullOrEmpty(path))
```

Comment:
I would reverse the order of the tests. No point in calling `Directory.Exists` for null or empty paths.

Reply:
done